tests/robot/suites/traffic/ip_route/test_ip_route.robot

```diff
@@ -77,3 +77,84 @@ First Configure Three Agents And Then Start Agents
     Ping From agent_vpp_1 To 20.1.1.2
     Ping From agent_vpp_2 To 20.1.1.2
     Ping From agent_vpp_3 To 10.1.1.2
+
+# Non default VRF table 2 used in Agent VPP Node agent_vpp_2
+Start Two Agents And Then Configure
+    [Setup]         Test Setup
+    [Teardown]      Test Teardown
+    Add Agent VPP Node    agent_vpp_1
+    Add Agent VPP Node    agent_vpp_2
+    Create Master memif0 on agent_vpp_1 with IP 10.1.1.1, MAC 02:f1:be:90:00:00, key 1 and m0.sock socket
+    Create Slave memif0 on agent_vpp_2 with IP 10.1.1.2, MAC 02:f1:be:90:00:02, key 1 and m0.sock socket
+    # to propagate configuration from etcd to vpp
+    sleep    1
+    # temporarily ip table add 2
+    vpp_term: Issue Command    node=agent_vpp_2    command=ip table add 2
+    # temporarily
+    vpp_term: Issue Command    node=agent_vpp_2    command=set int ip address del memif0/1 all
+    # temporarily set int ip table memif0/1 2
+    vpp_term: Issue Command    node=agent_vpp_2    command=set int ip table memif0/1 2
+    # temporarily
+    vpp_term: Issue Command    node=agent_vpp_2    command=set int ip address memif0/1 10.1.1.2/24
+
+    # try ping
+    Ping From agent_vpp_1 To 10.1.1.2
+    # this does not work for non default vrf: Ping From agent_vpp_2 To 10.1.1.1
+    Ping On agent_vpp_2 With IP 10.1.1.1, Source memif0/1
+
+# Non default VRF table 2 used in Agent VPP Node agent_vpp_2
+# Non default VRF table 3 used in Agent VPP Node agent_vpp_3
+Start Three Agents, Then Configure And Move Interfaces To Non Default VRF
+    [Setup]         Test Setup
+    [Teardown]      Test Teardown
```

Comment:
add KW for vrf configuration and manipulation in vpp_agent, to replace direct vpp configuration